app/test/e2e/utils/helpers.js

```diff
@@ -329,6 +340,18 @@ const mockDeleteGraphAssociation = (datasetId, application = 'rw', mockSuccess =
         .reply(mockSuccess ? 200 : 404, { data: mockSuccess ? {} : { message: 'Resource XXXX not found.' } });
 };
 
+const mockAddFavouriteResourceToGraph = (resourceType, resourceId, loggedUser) => {
+    nock(process.env.GATEWAY_URL)
+        .post(`/v1/graph/favourite/${resourceType}/${resourceId}/${loggedUser.id}`,)
```

Comment:
no body to validate?

Reply:
I'm not sure about what the body should have. Nor the favourites or graph documentation in doc-api, nor the request for graph api inside favourite.router shows anything about what this body is supposed to return. The request is done but without expecting nothing in return or use the body for anything inside the code.

Is this how it is supposed to work, though?